.drone/drone.jsonnet

```diff
@@ -0,0 +1,10 @@
+local triggers = import 'lib/drone/triggers.libsonnet';
+local pipeline = (import 'lib/drone/drone.libsonnet').pipeline;
+local withInlineStep = (import 'lib/drone/drone.libsonnet').withInlineStep;
+
+[
+  pipeline('ci')
+  + withInlineStep('lint', ['make lint'], image='grafana/jsonnet-build:7f1cb84')
```

Comment:
I figured out how we build this Docker image, so once we merge this I would send a follow-up PR to build it in Drone and auto update this Drone config.